src/util.ts

```diff
@@ -688,6 +689,17 @@ export const getIncreasedPriceFromExisting = (
   return getIncreasedPriceHex(convertPriceToDecimal(value), rate);
 };
 
+export const checkGasValues = (gasValues: GasValues) => {
+  const keys = Object.keys(gasValues);
+  keys.forEach((key) => {
+    const val = (gasValues as any)[key];
+    if (typeof val !== 'string' || !isHexString(val)) {
```

Comment:
i guess technically this `typeof` check is redundant since you can't set any of these to `any` (they're all `string`). That said, I don't think it hurts any to have an additional check.